tests/test_openhands_conversation_export_scripts.py

```
@@ -0,0 +1,136 @@
+                "text": f"Authorization: Bearer {token}",
+            },
+            "long": "x" * 160,
+        }
+
+        with tempfile.TemporaryDirectory() as td:
+            td_path = Path(td)
+            input_path = td_path / "input.json"
+            output_path = td_path / "output.json"
+            input_path.write_text(json.dumps(payload), encoding="utf-8")
+
+            subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)